docs/geedocs/5.3.0/answer/176738.html

```diff
@@ -255,31 +255,40 @@ <h2 id="cache_configuration">Cache configuration</h2>
 (assuming your asset root is <code>/gevol/assets</code>; if not, modify the path accordingly).
 The asset cache size is controlled by <code>AssetCacheSize</code> and the asset version cache size is controlled by <code>VersionCacheSize</code>.
 Note that the asset version cache is generally larger than the asset cache because it may contain multiple versions of the same asset.</p>
-<br><br>
-Within <code>misc.xml</code>, the following options are deprecated and are treated as if they are permanently enabled: <code>DisablePacketLevelVersionCachePurge</code> and <code>ConsolidateListenerNotifications</code>.
-<br><br>
-In some instances, memory utilization may become problematic. An experimental feature is included which will allow for modifying aspects of the the XML cache.  This is controlled through <code>/etc/opt/google/XMLparams</code>. It takes the form of a standard configuration file, and it is in the following format.
-<br><br>
+
+<p>Within <code>misc.xml</code>, the following options are deprecated and are treated as if they are permanently enabled: <code>DisablePacketLevelVersionCachePurge</code> and <code>ConsolidateListenerNotifications</code>.</p>
+
+<p>The Xerces library maintains a separate cache for dealing with XML files. An experimental feature is included which will allow for modifying aspects of this cache.
+This is controlled through <code>/etc/opt/google/XMLparams</code>. It takes the form of a standard configuration file, and it is in the following format.</p>
```

Comment:
It might be worth mentioning that this file needs to be created by the user if they want to use these options.